packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/nodeexpress/whatyouneed.md

```diff
@@ -0,0 +1,3 @@
+* [Set up your Okta org for a password optional use case](/docs/guides/oie-embedded-common-org-setup/android/main/#set-up-your-okta-org-for-a-password-optional-use-case)
+* [The Embedded SDK added to your app](/docs/guides/oie-embedded-common-download-setup-app/nodejs/main/)
```

Comment:
I changed it because I tried to make it consistent with "What you need". As in you need "The Embedded SDK added to your app".  I've changed it to be in compliance with the style guide but I think there should be exceptions.